# $OpenBSD: patch-ak,v 1.1.1.1 1999/02/05 05:05:19 fgsch Exp $
#
# This patch is benign when not using RSAREF and is required when using
# RSAREF to get pgp5 to link.  Without the patch pgp5 fails to link
# as NN_ModExp is defined in two libraries.
#
--- config.h.in.~1~	Sat Feb  7 11:54:56 1998
+++ config.h.in	Tue Feb  2 22:54:29 1999
@@ -69,7 +69,7 @@
 #endif
 
 #ifndef USE_MPI_LIB
-#define USE_MPI_LIB 1
+#define USE_MPI_LIB 0
 #endif
 
 /*The licensing level to include.  1 is Freeware, 2 is Beta, 3 is Full*/
